src/usr/share/gpgmailer/gpgkeyring.py

```diff
@@ -44,7 +44,9 @@ def __init__(self, gnupg_home):
         self.fingerprint_to_key_dict = {}
 
         for key in self.gpg.list_keys():
-            
+
+            email_list = []
```

Comment:
Move this closer to where you first use it.

Reply:
k